.github/workflows/compliance.yml

```diff
@@ -41,35 +41,21 @@ jobs:
 
     - name: Install python dependencies
       run: |
-        pip3 install setuptools
-        pip3 install wheel
-        pip3 install python-magic junitparser gitlint pylint pykwalify
-        pip3 install west
+        ./scripts/ci/ci_compliance.sh check_compliance python-deps
```

Comment:
This step is specific to GitHub ubuntu-20.04 virtual environment image (installing only what is not installed in the image). There is no point in moving this out to an external script.

Please use `requirements.txt` in your downstream CI.